inc/saiport.h

```diff
@@ -1498,6 +1501,71 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_QOS_TC_AND_COLOR_TO_MPLS_EXP_MAP,
 
+    /**
+     * @brief Port Down Error Status
+     *
+     * @type sai_port_err_status_list_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_ERR_STATUS_LIST,
+
+    /**
+     * @brief Fabric port Attached
+     *
+     * Signifies if the fabric port is attached
+     *
+     * @type bool
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_FABRIC_ATTACHED,
+
+    /**
+     * @brief Attached Switch type.
+     *
+     * Signifies the attached switch type
+     *
+     * @type sai_switch_type_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_FABRIC_ATTACHED_SWITCH_TYPE,
```

Comment:
Are the ATTACHED_SWITCH_TYPE, ATTACHED_SWITCH_ID etc attributes also available in FABRIC_REACHABILITY in the fabric_port_reachability_t structure?

Reply:
*ATTACHED* attributes are used to check for the device/port directly connected to a local fabric port.  If checking on a local VoQ switch, the connected device could be another VoQ switch (back to back) or a fabric switch.   On a local fabric switch, the connected device is most likely a VoQ switch but certain multi-level fabric topologies could connect fabric switches to other fabric switches.

*REACHABILITY* attribute is used to check if there is fabric connectivity from a local VoQ switch to a remote VoQ switch (regardless of which fabric port the cells traverse through).